Modules/IO/TransformInsightLegacy/test/itkIOTransformTxtTest.cxx

```diff
@@ -201,7 +201,10 @@ int
 secondTest(const std::string & outputDirectory)
 {
   std::filebuf fb;
-  fb.open((outputDirectory + "IllegalTransform.txt"), std::ios::out);
+  if (!fb.open((outputDirectory + "IllegalTransform.txt"), std::ios::out))
+  {
+    return EXIT_FAILURE;
+  }
   std::ostream os(&fb);
```

Comment:
Looks like this code would have been easier with _just_ `std::ofstream` (instead of `std::filebuf` + `std::ostream`) 🤔 

But no problem, your proposed adjustment look OK to me. 👍